src/Components/Alert/Components/Filters/Medium.tsx

```diff
@@ -0,0 +1,76 @@
+import { Checkbox, Column, GridColumns, Spacer, Text } from "@artsy/palette"
+import { FC } from "react"
+import { useAlertContext } from "Components/Alert/Hooks/useAlertContext"
+import { MEDIUM_OPTIONS } from "Components/ArtworkFilter/ArtworkFilters/MediumFilter"
+import { useArtworkFilterContext } from "Components/ArtworkFilter/ArtworkFilterContext"
+import { sortResults } from "Components/ArtworkFilter/ArtworkFilters/Utils/sortResults"
+import { intersection } from "lodash"
+import {
+  INITIAL_ITEMS_TO_SHOW,
+  ShowMore,
+} from "Components/Alert/Components/Filters/ShowMore"
+
+export const Medium: FC = () => {
+  const { state, dispatch } = useAlertContext()
+  const filters = useArtworkFilterContext()
+
+  const additionalGeneIDs = state.criteria.additionalGeneIDs || []
+
+  const toggleSelection = (selected, name) => {
+    let updatedValues = state.criteria.additionalGeneIDs || []
+
+    if (selected) {
+      updatedValues = [...updatedValues, name]
+    } else {
+      updatedValues = updatedValues.filter(item => item !== name)
+    }
+
+    dispatch({
+      type: "SET_CRITERIA_ATTRIBUTE",
+      payload: { key: "additionalGeneIDs", value: updatedValues },
+    })
+  }
+
+  const mediums = filters.aggregations?.find(agg => agg.slice === "MEDIUM") || {
+    slice: "",
+    counts: [],
+  }
+
+  const allowedMediums = mediums?.counts?.length
+    ? mediums.counts
+    : MEDIUM_OPTIONS
+
+  const intersected = intersection(
+    additionalGeneIDs,
+    allowedMediums.slice(INITIAL_ITEMS_TO_SHOW).map(({ value }) => value)
+  )
+  const hasBelowTheFoldMediumFilter = intersected.length > 0
+  const resultsSorted = sortResults(additionalGeneIDs, allowedMediums)
```

Comment:
For now, I have implemented the same sorting logic as the one used in the filter panel. I have opened a [Slack thread](https://artsy.slack.com/archives/C05EQL4R5N0/p1699277137885669) to clarify on whether we should move the selected options to the top or not.

Reply:
I will update this to not move selected options to the top, as discussed on [Slack](https://artsy.slack.com/archives/C05EQL4R5N0/p1699277137885669).